src/application/previewheaderbar.cpp

```diff
@@ -0,0 +1,48 @@
+// PDF Slicer
+// Copyright (C) 2017-2020 Julián Unrrein
```

Comment:
For files you create, you are the copyright holder! Put your name here, and the current year. Same thing in previewheader.hpp.

Reply:
As for the code you implemented, it's just what was needed. Good job :+1:

Reply:
> As for the code you implemented, it's just what was needed. Good job +1
My pleasure.